roles/run_hook/molecule/default/prepare.yml

```diff
@@ -33,6 +34,7 @@
       ansible.builtin.template:
         dest: "/tmp/{{ item }}"
         src: "dummy.yml.j2"
+        mode: "0775"
```

Comment:
why is this one 0775 what is the difference from the task above in this file which is 0644